addons/purchase/migrations/11.0.1.2/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,42 @@
+---Fields in module 'purchase'---
+purchase     / procurement.rule         / action (False)                : NEW selection_keys: ['buy', 'manufacture', 'move'], mode: modify
+# nothing to do as the selection key "buy" was already added in function _get_action before and is moved now directly to the field
+
+purchase     / purchase.order           / activity_ids (one2many)       : NEW relation: mail.activity
+# nothing to do as this was added by mail.activity.mixin, activities didn't exist before so that we have nothing to migrate
+
+purchase     / purchase.order           / website_message_ids (one2many): NEW relation: mail.message
+# nothing to do as this field was added by mail.thread in module portal and the field didn't exist before
+
+purchase     / purchase.order.line      / move_dest_ids (one2many)      : NEW relation: stock.move
+purchase     / stock.move               / created_purchase_line_id (many2one): NEW relation: purchase.order.line
+# nothing to do as these fields are filled by purchase.requisition which is new in v11
+
+purchase     / purchase.order.line      / orderpoint_id (many2one)      : NEW relation: stock.warehouse.orderpoint
+# nothing to do
```

Comment:
Same here. Migrate the orderpoint_id from the related procurement

Reply:
Same here. I updated the values now form procurement.order.